src/Contracts/NetworkResults.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Meilisearch\Contracts;
+
+class NetworkResults extends Data
+{
+    /**
+     * @var string the identifier for the local node
+     */
+    private string $self;
+
+    /**
+     * @var array<string, array{url: string, searchApiKey: string}> a mapping of remote node IDs to their connection details
+     */
+    private array $remotes;
+
+    /**
+     * @param array{
+     *     self?: string,
+     *     remotes?: array<string, array{url: string, searchApiKey: string}>
```

Comment:
are empty string here allowed?
```suggestion
     *     self?: non-empty-string,
     *     remotes?: array<non-empty-string, array{url: non-empty-string, searchApiKey: string}>
```